src/MizarWidgets/SamplingWithFrameTrackWidget.cpp

```diff
@@ -32,19 +41,28 @@ void SamplingWithFrameTrackWidget::Init(
 
 SamplingWithFrameTrackWidget::~SamplingWithFrameTrackWidget() = default;
 
-[[nodiscard]] Baseline<SamplingWithFrameTrackInputWidget*>
-SamplingWithFrameTrackWidget::GetBaselineInput() const {
+Baseline<SamplingWithFrameTrackInputWidget*> SamplingWithFrameTrackWidget::GetBaselineInput()
+    const {
   return Baseline<SamplingWithFrameTrackInputWidget*>(ui_->baseline_input_);
 }
 
-[[nodiscard]] Comparison<SamplingWithFrameTrackInputWidget*>
-SamplingWithFrameTrackWidget::GetComparisonInput() const {
+Comparison<SamplingWithFrameTrackInputWidget*> SamplingWithFrameTrackWidget::GetComparisonInput()
+    const {
   return Comparison<SamplingWithFrameTrackInputWidget*>(ui_->comparison_input_);
 }
 
-const Baseline<QString> SamplingWithFrameTrackWidget::kBaselineTitle =
-    Baseline<QString>(QStringLiteral("Baseline"));
-const Comparison<QString> SamplingWithFrameTrackWidget::kComparisonTitle =
-    Comparison<QString>(QStringLiteral("Comparison"));
+void SamplingWithFrameTrackWidget::OnSignificanceLevelSelected(int index) {
+  constexpr int kIndexOfFivePercent = 0;
+  significance_level_ =
+      (index == kIndexOfFivePercent) ? kDefaultSignificanceLevel : kAlternativeSignificanceLevel;
+}
+
+void SamplingWithFrameTrackWidget::OnMultiplicityCorrectionCheckBoxClicked(int state) {
+  is_multiplicity_correction_enabled_ = (state == Qt::CheckState::Checked);
```

Comment:
I assume you have this `is_multiplicity_correction_enabled_` field, to use it in the future (currently its a bit unnecessary. An alternative could be a method, something like this:
```
bool SamplingWithFrameTrackWidget::IsMultiplicityCorrectionEnabled() {
  return ui_->multiplicity_correction_->IsChecked();
}
```
Then you don't have this extra state and you dont need to make sure they are always in sync

Reply:
Yes, you are right. About how necessary this field will be. I'd prefer to keep it for now, as its elimination doesn't remove the need for `OnMultiplicityCorrectionCheckBoxClicked` slot.